pocs/utils/light_control.py

```diff
@@ -0,0 +1,141 @@
+#Basic Python interface for the qhue system.... ip for bridge -> "192.168.0.2"
+import qhue
+from os import path
+from qhue import Bridge, QhueException, create_new_username 
+
+#Setting Up Remote Access to Hue Bridge
+
+def ManualLogin(LED_State, Desk_State, ip):
+    
+    """
+    A way to manually login to the hue bridge, requires the button on the bridge to be pushed.
+    Only useful should auto login fail.
+    
+    LES_State, Desk_State can either be True or False to test the lights
+    
+    ip for current bridge is 192.168.0.2
+    
+    """
+    username = qhue.create_new_username(ip)
+    #double check the user name - can be removed later
+    print("The username created is", username)
+    
+    b = qhue.Bridge(ip,username)
+    
+    b.lights[1].state(bri=250,hue=30000,on=LED_State)
+    b.lights[4].state(bri=250,hue=10000,on=Desk_State) 
+    
+    return()
```

Comment:
Here you could return `b` so someone can manipulate the bridge using other functions or even their own function.